supported_interpreters.md

```diff
@@ -95,13 +99,26 @@ Please check the below table before you download Zeppelin package.
       <a href="http://tajo.apache.org/" target="_blank">Tajo</a>
       are available
     </td>
+    <td>
+      <a href="http://www.postgresql.org/" target="_blank">PostgreSQL</a>,
+      <a href="https://www.mysql.com/" target="_blank">MySQL</a>,
+      <a href="https://mariadb.org/" target="_blank">MariaDB</a>,
+      <a href="https://aws.amazon.com/documentation/redshift/" target="_blank">Redshift</a>,
+      <br/>
```

Comment:
I just copy and paste it from [other line](https://github.com/apache/zeppelin/pull/2209/files/3c836008ba81dc1216da3c7bc259ea9f92c14ff3#diff-b4f569e0e53323a3f20dc62535bfac06R95). I guess its for readability. Left cell is without `<br/>` right cell is with `<br/>`
![screen shot 2017-03-31 at 5 28 54 pm](https://cloud.githubusercontent.com/assets/8503346/24542706/9f7998ae-1637-11e7-9ec9-49dbe064d523.png)